model/commitqueue/commit_queue.go

```diff
@@ -52,6 +54,7 @@ func (q *CommitQueue) Enqueue(item CommitQueueItem) (int, error) {
 		return position + 1, errors.New("item already in queue")
 	}
 
+	item.EnqueueTime = time.Now()
```

Comment:
perhaps it would be informative to log when we enqueue an item. Some useful info could be:
- the item id
- project id
-  the queue length when we enqueue an item

If you think this is a good idea and we log when we enqueue and dequeue, maybe it's not necessary to store the enqueue time in an item.

Reply:
(the duration can be derived with Splunk [transactions](https://docs.splunk.com/Documentation/Splunk/7.3.1/SearchReference/Transaction))

Reply:
a second alternative is to calculate the time since the patch was created.

Reply:
Re: conversation, the Splunk transactions are interesting but we should probably log the duration in queue how we log other things and not introduce something new here. Keeping it this way for now!